aptos/token_bridge/sources/complete_transfer.move

```diff
@@ -218,7 +218,7 @@ module token_bridge::complete_transfer_test {
         deployer = @deployer,
         token_bridge = @token_bridge,
     )]
-    #[expected_failure(abort_code = 65542)] // EINSUFFICIENT_BALANCE
+    #[expected_failure(abort_code = 65542, location = 0000000000000000000000000000000000000000000000000000000000000001::coin)] // EINSUFFICIENT_BALANCE
```

Comment:
Is there a reason this isn't `0x1`?

Reply:
Interestingly, the error message from the compiler had it fully extended like..  unsure why tbh